core/neat/settings/_settings.scss

```diff
@@ -0,0 +1,20 @@
+/// Neat default grid.
+///
+/// @type map
+///
+/// @property {number (unitless)} columns 12
+///   Number of grid columns
+///
+/// @property {number (with unit)} gutter 20px
+///   Grid gutter width.
+///
+/// @property {number (unitless)} max-width 1200px
+///   Grid container's max-width.
+///
+/// @access private
+
+$neat-default-grid: (
+  columns: 12,
+  gutter: 20px,
+  max-width: 1200px,
```

Comment:
Yes I agree with you guys now. After spiking further, I found myself needing to change the max width at breakpoints. It quickly went down the rabbit hole, when it feels much more natural for the Developer user to do it in the pattern they are making. :+1: